client/lib/plans/personal-plan.js

```diff
@@ -0,0 +1,36 @@
+import { translate } from 'i18n-calypso';
+
+import { PLAN_PERSONAL } from './constants';
+
+export default {
+	product_id: 1009,
+	product_name: translate( 'WordPress.com Personal' ),
+	prices: {
+		USD: 71.88,
+		NZD: 105.35,
+		AUD: 99.05,
+		CAD: 94.05,
+		JPY: 7874.06,
+		EUR: 64.27,
+		GBP: 49.86
+	},
```

Comment:
I can safely remove every price except `USD` so unit tests pass. This is not used at all except for some tests I won't clean up.
Prices are calculated and formatted on the backend, and users will always see the Personal Plan price in `USD` until the Desktop App is updated 😕  and we can remove this code.